tests/unit-tests/name.t.cpp

```diff
@@ -405,8 +405,8 @@ BOOST_AUTO_TEST_CASE(Compare)
 {
   BOOST_CHECK_EQUAL( 0, Name("/A")  .compare(Name("/A")));
   BOOST_CHECK_EQUAL( 0, Name("/A")  .compare(Name("/A")));
-  BOOST_CHECK_EQUAL(-1, Name("/A")  .compare(Name("/B")));
-  BOOST_CHECK_EQUAL( 1, Name("/B")  .compare(Name("/A")));
+  BOOST_CHECK_GT(-1, Name("/A")  .compare(Name("/B")));
```

Comment:
hmm are you sure? it should be either `_GE(-1, ...)` or `_GT(0, ...)`, otherwise it'll break on x86{,_64}

Same thing for the other checks below.